app/forms/work_form.rb

```diff
@@ -4,4 +4,18 @@
 class WorkForm < ApplicationForm
   attribute :title, :string
   validates :title, presence: true
+  accepts_nested_attributes_for :related_links
+
+  def save!(user:)
+    Work.transaction do
+      # Setting the deposit_job_started_at to the current time to indicate that the deposit job has started and user
+      # should be "waiting".
+      Work.create!(title:, user:, deposit_job_started_at: Time.zone.now).tap do |work|
+        related_links.each do |related_link|
+          related_link.work = work
+          related_link.save!
+        end
+      end
+    end
+  end
```

Comment:
We need to roll our own save per the above. But I'm not even sure if we should be persisting related links in the app DB at all, so this likely needs more work.